src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/fields/Url.js

```diff
@@ -60,35 +60,34 @@ export default class Url extends React.Component<FieldTypeProps<?string>> {
                     value: defaults = [],
                 } = {},
                 schemes: {
-                    value: schemes = [
-                        {name: 'http://'},
-                        {name: 'https://'},
-                        {name: 'ftp://'},
-                        {name: 'ftps://'},
-                    ],
+                    value: schemes = undefined,
                 } = {},
             } = {},
             value,
         } = this.props;
 
-        if (!Array.isArray(schemes) || schemes.length === 0) {
-            throw new Error('The "schemes" schema option must contain some values!');
-        }
+        let protocols = undefined;
 
-        const protocols = schemes.map((scheme) => {
-            if (typeof scheme.name !== 'string') {
-                throw new Error(
-                    'Every schema in the "schemes" schemaOption must contain a string string name'
-                );
+        if (schemes) {
+            if (!Array.isArray(schemes) || schemes.length === 0) {
```

Comment:
we usually use something like `if (defaults !== undefined && (!Array.isArray(schemes) || schemes.length === 0)) {`

Reply:
the beneath part is not in the this if thats why the nesting is needed